.github/workflows/codeql.yml

```
@@ -33,15 +34,19 @@ jobs:
             matrix:
                 include:
                     - language: csharp
-                      build-mode: none
+                      build-mode: manual
                     - language: go
                       build-mode: autobuild
```

```suggestion
                      build-mode: manual
```
If you have `elif [ "${{ matrix.language }}" == "go" ]; then` in the manual building if-else section, you need to set `go` to manual here as well.

---

I actually need to remove from manual in the manual, ill check it. Thanks for noticing.